doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1772,10 +1779,69 @@ In order to save hard disk space and avoid loss of the audit log records during
 The X-Road software does not offer special tools for archiving the audit log. The *rsyslog* can be configured to redirect the audit log to an external location.
 
 
-## 13 Back up and Restore
+## 13 Backing up and Restoring Configuration
+
+It is possible to back up and later restore security server configuration.
+
+### 13.1 Contents of configuration backup archive
+
+Backup archive contains
+- copy of serverconf database
+- user modifiable configuration files
+- TLS keys and certificates (both internal and ui certificate)
+- database credentials
+
+Note that backups contain information that must be kept secret (especially TLS keys and database credentials).
+In other words leaking this information could easily lead to full compromise of security server.
+It is therefore highly recommended that backup archives are encrypted and stored securely. Should the information still 
+leak for whatever reason the security server should be considered as compromised and reinstalled from scratch.
+
+### 13.2 Backup archive encryption and verification
+
+Starting from version 7.0 security server backups are signed and optionally encrypted. The GNU Privacy Guard \[[GnuPG](#Ref_GnuPG)\] 
+is used for encryption and signing. GPG keypair is generated during security server initialisation. In addition to generated
+key additional public keys can be used to encrypt backups. 
+
+### 13.3 Backup encryption configuration
+
+Back up encryption is initially turned off. To turn encryption on configuration must be overridden in the file `/etc/xroad/conf.d/local.ini`, in the `[proxy]` section.
+
+    [proxy]
+    backup-encrypted=false
+    backup-public-key-path=/etc/xroad/backupkeys
 
+Backups are always signed. To turn on backup encryption add or change the `backup-encrypted` key to true. If needed also 
+change the folder where additional backup encryption keys are stored. Default is `/etc/xroad/backupkeys`.
 
-### 13.1 Back up and Restore in the User Interface
+Backup encryption keys folder must contain only GPG public key files. In addition to security servers public key backup
+archives are encrypted with all the keys in this folder. It is recommended to use at least one additional public key,
+otherwise the backups will be unusable in case security servers private key is lost. It is up to security servers administrator
+to check that private keys used are sufficiently strong, there are no automatic checks. Additional keys for backup 
+encryption should be generated and stored outside security server in a secure environment.
+
+### 13.4 Verifying backup archive consistency
+
+Security server verifies consistency of backup archives automatically. It is also possible to check archives externally.
```

Comment:
It would be good to clarify here when does the verification take place. When restore is executed and/or when a backup file is uploaded?